README.md

```diff
@@ -1,58 +1,14 @@
-[![CircleCI](https://circleci.com/gh/facebookincubator/Polygames.svg?style=svg)](https://circleci.com/gh/facebookincubator/Polygames)
```

Comment:
eventually it'd be good to get CI running again, but I guess there's not much to test (besides that the Docker image builds?) until we get the original polygames tests compiling again